src/lib/throttle.ts

```diff
@@ -0,0 +1,20 @@
+import { throttle as _throttle } from 'lodash';
```

Comment:
The `_` prefix in typescript means to not warn if it's unused so it's better not to use it except for that specific case:
```suggestion
import { throttle as $throttle } from 'lodash';
```